frontend/src/lang/pl-PL.json

```
@@ -0,0 +1,94 @@
+{
+    "languageName": "Polish",
```

```suggestion
    "languageName": "Polski",
```
language names are typically in the language itself